documentation/tutorial-checklist.md

```diff
@@ -0,0 +1,35 @@
+---
+jupytext:
+  notebook_metadata_filter: myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
++++ {"user_expressions": []}
+
+# Tutorial checklist
+
+```{note}
+This guidelines also apply to [blog posts](https://ploomber.io/blog/)
+```
+
+## Sample data must be publicly available
+
+Datasets must be available publicly so we can download them inline. In other words, you should be able to get a URL that when opened should take you to the raw data ([example](https://raw.githubusercontent.com/mwaskom/seaborn-data/master/penguins.csv)). Furthermore, the data should be available in a third-party site and not uploaded to any of our repositories.
```

Comment:
Wonder how we can incorporate the following links as examples

 https://archive.ics.uci.edu/ml/index.php

https://www.nyc.gov/site/tlc/about/tlc-trip-record-data.page

@edublancas  can add them along with a mini sample script to extract them via requests + pandas

Reply:
Thoughts?